ssl/ssl_lib.c

```diff
@@ -4455,3 +4460,23 @@ int ssl_log_master_secret(SSL *ssl,
                           master_len);
 }
 
+int ssl_log_secret(SSL *ssl,
+                   const char *label,
+                   const uint8_t *client_random,
+                   size_t client_random_len,
+                   const uint8_t *secret,
+                   size_t secret_len)
+{
+    if (client_random_len != 32) {
+        SSLerr(SSL_F_SSL_LOG_MASTER_SECRET, ERR_R_INTERNAL_ERROR);
+        return 0;
+    }
+
+    return nss_keylog_int(label,
+                          ssl,
+                          client_random,
+                          client_random_len,
+                          secret,
+                          secret_len);
+}
```

Comment:
This function is basically identical to the one above it, except that the one above doesn't fire for TLSv1.3. I suspect we should collapse these two and hoist up the check for TLSv1.3 into the calling scope of the above function, but I wanted a code reviewer to confirm that was a good idea.

An alternative would be to have the function above call this one: either would work.